man/kernel-install.xml

```diff
@@ -497,6 +497,25 @@
           <xi:include href="version-info.xml" xpointer="v198"/>
           </listitem>
       </varlistentry>
+      <varlistentry>
+        <term>
+          <filename>/usr/lib/kernel/devicetree</filename>
+          <filename>/etc/kernel/devicetree</filename>
+        </term>
+          <listitem>
+            <para>Read by <filename>90-loaderentry.install</filename>. The content of the file
+            <filename>/etc/kernel/devicetree</filename> specifies the device tree blob to install with the
```

Comment:
"device tree blob" → "filename of the compiled Devicetree"?

you never say that this is a file name